client/hbs/post.hbs

```diff
@@ -1,55 +1,46 @@
 <article>
 	{{#if title.url}}
-	<h1><a class="no-display" href="{{title.url}}">{{title.text}}</a></h1>
+		<h1><a class="no-display" href="{{title.url}}">{{title.text}}</a></h1>
```

Comment:
Clicking on any part of the title bar and having it link is awkward imo, especially when the title is short compared to the title bar width.